mlflow/genai/judges/instructions_judge/__init__.py

```diff
@@ -147,6 +151,11 @@ def template_variables(self) -> set[str]:
         """Get the template variables from the instructions."""
         return self._instructions_prompt.variables
 
+    @property
+    def is_multi_turn(self) -> bool:
```

Comment:
nit: let's be consistent with terminology in the code. Right now there's `multi-turn`, `session`, and `conversation`. My preference is to use `session` in the code and we can use `multi-turn`/`conversation` in user-facing APIs/docs (e.g., the `{{ conversation }}` param).